app/src/community/java/com/twilio/video/app/data/api/AuthServiceRequestDTO.kt

```diff
@@ -4,5 +4,6 @@ data class AuthServiceRequestDTO(
     val passcode: String? = null,
     val user_identity: String? = null,
     val room_name: String? = null,
-    val create_room: Boolean? = false
+    val create_room: Boolean? = true,
```

Comment:
The default value of `create_room: Boolean` changed to `true`. Is that intenetional?

Reply:
Good question! This DTO object can be ignored for now. On this branch, this change along with the chat UI button code are used to test out the conversation development back end quick deploy server that @timmydoza is working on along with the `ChatManagerImpl` in the app. In one of the UI stories I'll finalize this logic and add some tests behind it.

Reply:
But good call out though. In the state that this logic is in, it should not yet be merged to master until the `feature/conversations` is complete